packages/frontend/Caddyfile

```diff
@@ -0,0 +1,14 @@
+{$SERVER} {
+  handle /api* {
+		reverse_proxy api:{$API_PORT}
+	}
+  handle /uploads* {
+		reverse_proxy api:{$API_PORT}
```

Comment:
Whats the point of splitting up /api and /uploads if they're both just reverse proxies to the express backend? Why not just send all traffic to the express backend and let it do the routing?

Reply:
What you mean?

Everything that is not `/api` and `/uploads` is the frontend and being directly served by Caddy.

One option though would be to let Express serve the frontend as well and have one less Docker image in total.

Reply:
Ah right okay. Another options would be to use two seperate Caddy hosts: one that is *just* a reverse proxy to the express server, and another that *just* serves the static frontend. That would allow us to define the frontend and api domains independently.

But I think this approach is fine.